liquibase-standard/src/main/java/liquibase/database/core/MySQLDatabase.java

```diff
@@ -273,6 +271,25 @@ public int getDatabasePatchVersion() throws DatabaseException {
 
     }
 
+    public Integer getTimeStampFSPWith(int columnSize) {
```

Comment:
I think this could be improved in terms of efficiency. I propose adding an "if" statement and then just returning columnSize%10.

It would be something along the lines of:
```
public Integer getTimeStampFSPWith(int columnSize) {
    if (columnSize > 20 && columnSize < 27) {
        return columnSize % 10
    }
    return 0
}
```

I'm unaware of the lower and upper limits on the columnSize, so it might not need both verifications.

Reply:
Thank you very much for your advice.
I tested the method and confirmed that it works correctly.
After changing to this method, I proceeded to commit again.